include/deal.II/base/bounding_box.h

```diff
@@ -140,6 +140,16 @@ class BoundingBox
   bool
   point_inside(const Point<spacedim, Number> &p) const;
 
+  /**
+   * Increase (or decrease) the size of the bounding box by the given amount.
+   *
+   * If you call this method with a negative number, and one of the axes of the
+   * original bounding box is smaller than amount/2, the method will trigger
+   * an assertion.
```

Comment:
I didn't see this, but it would be useful to say somewhere that you're growing the bounding box in *both* directions in *all* coordinate directions. This wasn't clear to me from the description.